sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -1621,7 +1621,7 @@ var setParentClass = function(child, parent) {
     };
 
     glsBuiltinPrecisionTests.InverseSqrt.prototype.precision = function(ctx, ret, x) {
-        if (x < 0)
+        if (x <= 0)
```

Comment:
The native code: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/glshared/glsBuiltinPrecisionTests.cpp#2076